dts/arm/nxp/nxp_mcxn94x_common.dtsi

```diff
@@ -534,6 +535,36 @@
 		interrupts = <57 0>;
 		status = "disabled";
 	};
+
+	lpadc0: lpadc@10D000 {
```

Comment:
register style is lowercase, IE `10dd00`. Same thing applies to the `reg` property.

Reply:
Thanks for your reminder, I will change it to lowercase.